adding_species_manuscript.tex

```diff
@@ -333,7 +333,7 @@ \section*{The utility of \stdpopsim for genome-wide simulations}
 an even stronger effect. Selection acting on a small number of sites can
 indirectly influence levels and patterns of genetic variation at linked neutral sites,
 which has been shown to have a widespread
-effect on patterns of genome variation in myriad species
+effect on patterns of genome variation in a myriad of species
```

Comment:
Either of these are grammatically correct, the first way ("in myriad species") is similar to saying "in many species," the second way ("in a myriad of species") is similar to "in a lot of species." Coin flip?

Reply:
I leave it up to you to decide....